service/chartconfig/v1/helm/helm.go

```diff
@@ -46,21 +47,30 @@ func New(config Config) (*Client, error) {
 	return newHelm, nil
 }
 
-// GetReleaseContent gets the current status of the Helm Release.
+// GetReleaseContent gets the current status of the Helm Release including any
+// values provided when the chart was installed.
 func (c *Client) GetReleaseContent(customObject v1alpha1.ChartConfig) (*Release, error) {
 	releaseName := key.ReleaseName(customObject)
 
 	resp, err := c.helmClient.ReleaseContent(releaseName)
-	if IsReleaseNotFound(err) {
+	if err != nil && IsReleaseNotFound(err) {
 		return nil, microerror.Maskf(releaseNotFoundError, releaseName)
 	}
 	if err != nil {
 		return nil, microerror.Mask(err)
 	}
 
+	// Raw values are returned by the Tiller API and we convert these to a map.
+	raw := []byte(resp.Release.Config.Raw)
+	values, err := chartutil.ReadValues(raw)
+	if err != nil {
+		return nil, microerror.Mask(err)
+	}
+
```

Comment:
We need this foo to get the values. We use the Helm `chartutil` package rather than doing it ourselves.